components/tag-list/demo/tag-list.html

```diff
@@ -9,6 +9,18 @@
 			import '../tag-list.js';
 			import '../tag-list-item.js';
 			import '../test/tag-list-item-mixin-consumer.js';
+
+			window.addEventListener('load', () => {
```

Comment:
I _think_ this could be simplified:
```javascript
document.addEventListener('d2l-tag-list-item-cleared', e => {
   e.target.deleteItem();
   ...
});
```

If the script needs to be specific to a snippet, it could also be embedded in fixture instead of outside.

Reply:
Usually we would place such script at the very bottom of the page, just before the page `body`, to avoid the script blocking the page rendering, but in this case it's not a huge deal.